pyext/libc.c

```diff
@@ -264,8 +264,8 @@ func_regex_first_group_match(PyObject *self, PyObject *args) {
   }
 
   // Assume there is a match
-  regoff_t start = m[1].rm_so;
-  regoff_t end = m[1].rm_eo;
+  regoff_t start = m[0].rm_so;  // Why was this 1 before?
```

Comment:
Any idea why this was `m[1]` before? I ran into an issue where `m` would always equal `(-1, -1)` because `m[1]` was always unset (default is `(-1, -1)`, [ref](https://linux.die.net/man/3/regexec): "Any unused structure elements will contain the value -1.")